test/functional/tests/lazy_writes/test_lazy_writes_clean.py

```diff
@@ -231,86 +231,119 @@ def test_clean_remove_core_with_fs(cache_mode, fs):
         remove(mnt_point, True, True, True)
 
 
+@pytest.mark.parametrize("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_clean_remove_core_without_fs(cache_mode):
+    """
+        title: Test of the ability to remove core from cache in modes with lazy writes.
+        description: |
+          Test if OpenCAS removes core with in modes with lazy writes without data loss.
+        pass_criteria:
+          - Core removing works properly.
+          - Data on core device is correct after core is removed.
+    """
+    with TestRun.step("Prepare devices for cache and core."):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(256, Unit.MebiByte)])
+        cache_part = cache_dev.partitions[0]
+        core_dev = TestRun.disks['core']
+        core_dev.create_partitions([Size(512, Unit.MebiByte)])
+        core_part = core_dev.partitions[0]
+        Udev.disable()
+
+    with TestRun.step(f"Start cache in {cache_mode} mode."):
+        cache = casadm.start_cache(cache_part, cache_mode)
+
+    with TestRun.step("Add core to cache."):
+        core = cache.add_core(core_part)
+        # get IO stats independent from test
         core_disk_writes_diff = IOstat.get_iostat(core_part).total_writes.value
```

Comment:
change name - this is not a diff (unless you treat 0 as the value for differentiation)